### PR TITLE
Add option for message formatting (-- to ‏‎−, >> to », << to «) in "Settings" window

### LINKED ISSUE #522: Option for message formatting (-- to ‏‎−, >> to », << to «)
Hi.
I want to report a bug in the telegram desktop version. (Linux)
The problem of the character: Dash. Or (-)
When two characters Dash, taken together, the result rather than the (--) becomes a long line. (‏‎−)
But when the two characters together Dash, the text first, this problem does not exist.
For example:
http://s4.picofile.com/file/8173777318/snapshot.png

### PR DESCRIPTION
**Pull request by PeterMX**
this closes #522  because add an option in settings to disable message formatting, it's enable by default so it's not a problem for people than use it.

### CODE REVIEW

File: Telegram/Resources/langs/lang.strings [hunk 1]
```diff
@@ -296,6 +296,7 @@ https://github.com/telegramdesktop/tdesktop/blob/master/LEGAL
 
 "lng_settings_section_chat_settings" = "Chat Settings";
 "lng_settings_replace_emojis" = "Replace emoji";
+"lng_settings_message_formatting" = "Message formatting";
```
Comment (stek29) [inline Telegram/Resources/langs/lang.strings]: "message formatting" isn't clear enough imo

Comment (kirsan31) [reply Telegram/Resources/langs/lang.strings]: This _strange_ replacements are made by telegram, not clear enough too :)
I really can't understand for what\whom it's done, and therefore have strong feeling that this PR will be rejected...

Comment (PeterMX) [reply Telegram/Resources/langs/lang.strings]: someone knows how should it be called?

Comment (JuanPotato) [reply Telegram/Resources/langs/lang.strings]: replace double chars (<<, >>, --) ?

Comment (kirsan31) [reply Telegram/Resources/langs/lang.strings]: Or   
```
Double chars (<<, >>, --) replacement.
Double chars (<<, >>, --) to special chars.
Convert (<<, >>, --) to special chars.
```
But, imho, this _feature_ does not deserve an option in settings, and simple must be removed.

Comment (PeterMX) [reply Telegram/Resources/langs/lang.strings]: Should I wait for change request?

Comment (PeterMX) [reply Telegram/Resources/langs/lang.strings]: @stek29 It's clear enough now.

**Comment by stek29** [comment]
See 4b763a7

**Pull request closed without merging**